delivery_service.rb

```diff
@@ -1,2 +1,37 @@
+require_relative 'car'
+require_relative 'bike'
+require_relative 'vehicle_availabilities'
+
+
 class DeliveryService
-end
\ No newline at end of file
+  extend VehicleAvailabilities
+
+  def initialize
+    @cars = DeliveryService.availability
+  end
+
+  def vehicle(weight, distance)
```

Comment:
this method should be named like create_delivery or something more informative

Reply:
done